### PR TITLE
Fix Firebase class print

### PR DESCRIPTION
**Pull request by itkonen**

### CODE REVIEW

**Comment by itkonen** [comment]
I'm not quite sure if the test for a user being signed in is valid in all states (e.g. after sign-out or connection error).

File: R/class-auth.R [hunk 1]
```diff
@@ -41,11 +41,18 @@ FirebaseAuth <- R6::R6Class(
...
-      cli_alert_info(signin, "\n")
+      user <- self$get_signed_in()
+      if (!isTRUE(user$success)) {
+        cli_alert_info("No user is signed in")
+      } else {
+        cli_alert_info("A user is signed in")
+        cli::cli_dl(c(
+          uid = user$response$uid,
+          email = user$response$email,
+          emailVerified = user$response$emailVerified,
```
Comment (JohnCoene) [inline R/class-auth.R:52]: This class is used as parent for all other auth methods.
Some of these fields in the `user` object may not be available, e.g. `emailVerified`.
If we want clearer `print` methods we can override in children classes.

**Review by JohnCoene — CHANGES_REQUESTED** [review]

**Comment by JohnCoene** [comment]
I did not work on the `print` methods too much because being shiny-dependent it tends not to be printed

**Comment by itkonen** [comment]
I included a `Filter` function in f0db083339c013fc56f7137f285b8312ab1557ed to clean up fields which might be missing. I guess this should also work for the children classes. Also, it would easy to add other fields that the children classes might have.

**Comment by JohnCoene** [comment]
Thank you, that works!

**Pull request merged**